### PR TITLE
Check mainchain tx confirmation in ante

### PR DESCRIPTION
**Pull request by jdkanani**

### CODE REVIEW

**Comment by jdkanani** [comment]
Fixes #142

File: auth/ante.go [hunk 1]
```diff
@@ -55,12 +56,21 @@ type FeeCollector interface {
 	) sdk.Error
 }
 
+//
+// MainTxMsg tx hash
+//
+type MainTxMsg interface {
```
Comment (vaibhavchellani) [inline auth/ante.go]: Please use a better name for this struct.

Comment (vaibhavchellani) [reply auth/ante.go]: Suggestion: `BaseMsg`. Maybe we should make all our `msg` satisfy `BaseMsg` which would satisfy `sdk.Msg` as well.

Comment (jdkanani) [reply auth/ante.go]: `BaseMsg` shows the wrong terminology here. This tx should satisfy the Main chain tx.

**Pull request merged**